src/dex/pancakeswap-v3/constants.ts

```diff
@@ -0,0 +1,35 @@
+export const PANCAKESWAP3_FUNCTION_CALL_GAS_COST = 21_000; // Ceiled
+export const PANCAKESWAPV3_TICK_GAS_COST = 24_000; // Ceiled
```

Comment:
I think it should be adapted to recent UniV3 changes

Reply:
@Verisana updated